engine-tests-connector/src/connector.rs

```diff
@@ -1311,6 +1311,58 @@ async fn test_ft_metadata() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn test_storage_deposit() -> anyhow::Result<()> {
+    let contract = TestContract::new().await?;
+    contract.call_deposit_eth_to_near().await?;
+
+    let res = contract
+        .engine_contract
+        .call("storage_deposit")
+        .args_json(json!({
+            "account_id": "account.near",
+        }))
+        .deposit(NearToken::from_yoctonear(1250000000000000000000))
+        .max_gas()
+        .transact()
+        .await?;
+    assert!(res.is_success(), "{res:#?}");
+
+    let res = contract
+        .engine_contract
+        .call("storage_balance_of")
+        .args_json(json!({
+            "account_id": "account.near",
+        }))
+        .max_gas()
+        .transact()
+        .await?
+        .json::<Value>()?;
+
+    assert_eq!(res, json!({"available": "0", "total": "0"}));
```

Comment:
Please add a comment about why 0 is the expected response when a non-zero amount of tokens were deposited. Something along the lines of "The NEP-141 implementation of ETH intentionally set the storage deposit amount equal to 0 so any non-zero deposit amount is automatically returned to the user, leaving 0 storage balance behind."